dashboard-app/src/components/Navbar/Navigation.tsx

```diff
@@ -22,7 +22,7 @@ const Link = styled(L)`
 
 const Navigation: React.FunctionComponent<Props> = (props) => (
   <Row between={'xs'}>
-    {['activity', 'volunteer', 'time'].map((page) =>
+    {['Activity', 'Volunteer', 'Time'].map((page) =>
```

Comment:
⏪ this breaks the navigation...

We should also add a clientside 404 page at some point